registration/include/pcl/registration/impl/ndt.hpp

```diff
@@ -48,24 +48,33 @@ NormalDistributionsTransform<PointSource, PointTarget>::NormalDistributionsTrans
 : target_cells_()
 , resolution_(1.0f)
 , step_size_(0.1)
-, outlier_ratio_(0.55)
+, outlier_ratio_(0.1)
 , gauss_d1_()
 , gauss_d2_()
 , trans_probability_()
 {
   reg_name_ = "NormalDistributionsTransform";
 
+  transformation_epsilon_ = 0.1;
+  max_iterations_ = 35;
+}
+
+template <typename PointSource, typename PointTarget>
+void
+NormalDistributionsTransform<PointSource, PointTarget>::setGaussianFittingParameters(
```

Comment:
Since we are breaking ABI, we can rethink on which parameters are class parameters, and which aren't. The non-class parameters (which change for every point) **need** to be extracted because there's already a PR to add OpenMP support which would mess things up if current situation continues